### PR TITLE
pom.xml fixed according to 21 Java

### PR DESCRIPTION
**Pull request by Y-Pazniak**
packages was extracted from the root
big Storage enum was divided into the small parts
constants was renamed into better (in my opinion) way into enums constans was divided into groups
main was extracted from the controller package and moved to the root createSummaryString was renamed

### CODE REVIEW

File: src/main/java/by/custom/utilcalculator/service/Bot.java [hunk 1]
```diff
@@ -196,23 +201,23 @@ private String getStringOptionsForEngineVolume() {
     private String getStringOptionsForAgeAuto() {
         return "." +
                 "\n" +
-                Storage.AGE_OF_AUTO;
+                Questions.AGE_OF_AUTO;
     }
 
     private String getStringOptionsForPrice() {
```
Comment (MaximPilipeiko) [inline src/main/java/by/custom/utilcalculator/service/Bot.java:207]: I proceed to see `String` in many methods. Is it really needed?

File: src/main/java/by/custom/utilcalculator/service/CalculatorPassenger.java [hunk 2]
```diff
@@ -26,10 +28,11 @@ public String calculate(CountryOrigin countryOrigin, OwnersType ownersType, Type
             case null -> {
                 return "calculator: countryOrigin is null";
             }
-            case default -> {
+            default -> {
                 return "calculator: calculate() unknown error";
             }
         }
+//        return "calculator: calculate() unknown error";
```
Comment (MaximPilipeiko) [inline src/main/java/by/custom/utilcalculator/service/CalculatorPassenger.java:35]: why not remove it?

**Review by MaximPilipeiko — APPROVED** [review]

File: src/main/java/by/custom/utilcalculator/service/Bot.java (hunk 1 above)
Comment (Y-Pazniak) [reply src/main/java/by/custom/utilcalculator/service/Bot.java:207]: We don't need it, sorry, my fault - didn't clean it yestarday. Today's fix contains less ...String... methods names. I still have two methods with this words, but I will try to fix it too.

File: src/main/java/by/custom/utilcalculator/service/CalculatorPassenger.java (hunk 2 above)
Comment (Y-Pazniak) [reply src/main/java/by/custom/utilcalculator/service/CalculatorPassenger.java:35]: I love this messages a lot. They really helped me when I changed Java-version, and it was hard for me to remove it. Anyway, new commit doesn't contain this comments. Sorry for this mess. I will try to put my code in better order.

**Pull request merged**